### PR TITLE
[Hub Generated] Review request for Microsoft.Capacity to add version preview/2018-06-01

### PR DESCRIPTION
**Pull request by gautam714**
If you are a MSFT employee you can view your [work branch via this link](https://ppe-openapihub.azure-devex-tools.com/branch/gautam714/azure-rest-api-specs/dev-reservations-Microsoft.Capacity-2018-06-01-riautorenewal...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by daschult** [comment]
@openapi sdkautomation rebuild

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/GetReservationDetails.json [hunk 1]
```diff
@@ -17,8 +17,10 @@
         "location": "eastus",
         "properties": {
           "appliedScopes": [
-            "/subscriptions/98df3792-7962-4f18-8be2-d5576f122de3"
+              "/subscriptions/98df3792-7962-4f18-8be2-d5576f122de3"
```
Comment (sergey-shandar) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/GetReservationDetails.json]: Please use 2 spaces for indentation.

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json [hunk 2]
```diff
@@ -357,13 +357,20 @@
...
+                "$ref": "#/parameters/ReservationIdParameter"
+            },
+            {
+                "$ref": "#/parameters/ReservationOrderIdParameter"
+            },
+            {
+                "$ref": "#/parameters/ApiVersionParameter"
+            },
+            {
+                "$ref" : "#/parameters/AppendRenewPropertiesParameter"
```
Comment (michaeljqzq) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: Please provide examples about this part (maybe other newly added properties as well)

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/GetReservationDetails.json (hunk 1 above)
Comment (gautam714) [reply specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/GetReservationDetails.json]: Done.

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json (hunk 2 above)
Comment (gautam714) [reply specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: Provided.

**Review by sergey-shandar — APPROVED** [review]

**Review by michaeljqzq — APPROVED** [review]

**Pull request merged**